vcloud_network_plugin/floatingip.py

```diff
@@ -129,7 +129,7 @@ def _floatingip_operation(operation, vca_client, ctx):
         if SSH_PUBLIC_IP in ctx.source.instance.runtime_properties:
             del ctx.source.instance.runtime_properties[SSH_PUBLIC_IP]
         if SSH_PORT in ctx.target.instance.runtime_properties:
-            del ctx.source.instance.runtime_properties[SSH_PORT]
+            del ctx.target.instance.runtime_properties[SSH_PORT]
```

Comment:
In code on one line above - checked  ctx.target.instance not ctx.source.instance